src/views/VenuesListWithFilter/FilterList/index.jsx

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from 'react'
+import styles from './styles.module.css'
+import { PriceRange } from './PriceRange'
+import { Dropdown } from './Dropdown'
+import { Checkboxes } from "./Checkboxes";
+
+const amenitiesFilters = [
+  'parking',
+  'WiFi',
+  'restaurant',
+  'bar',
+  'pool',
+  'jacuzzi',
+  'specjal'
+]
+
+const roomAmenitiesFilters = [
+  'kitchen facilities',
+  'bathroom facilities',
+  'air conditioning',
+  'safe',
+  'TV'
+]
+
+const neighbourhoodsFilters = [
+  'lake',
+  'forest',
+  'mountains',
+  'sea',
+  'river',
+  'park',
+  'zoo',
+  'national park'
+]
+
+export function FilterList () {
+  const [priceValue, setPriceValue] = useState([120, 370])
```

Comment:
```suggestion
const DEFAULT_MIN_PRICE = 120;
const DEFAULT_MAX_PRICE = 120;
export function FilterList () {
  const [priceValue, setPriceValue] = useState([DEFAULT_MIN_PRICE, DEFAULT_MAX_PRICE])
```
Try not to use "magic numbers".